src/SocialAuthUserManager.php

```diff
@@ -168,14 +170,14 @@ public function setSessionKeysToNullify(array $session_keys) {
    * @return \Symfony\Component\HttpFoundation\RedirectResponse
    *   A redirect response.
    */
-  public function authenticateUser($name, $email, $pluginId, $provider_user_id, $picture_url = FALSE, $data = '') {
+  public function authenticateUser($name, $email, $pluginId, $provider_user_id, $token, $picture_url = FALSE, $data = '') {
```

Comment:
I told you that $pluginId wasn't necessary here since the implementer's controller constructor already calls setPluginId()